3634/0/triangle.rb

```diff
@@ -0,0 +1,36 @@
+# Triangle Project Code.
+# rubocop:disable Metrics/CyclomaticComplexity
+# rubocop:disable Metrics/AbcSize, Lint/UnneededCopDisableDirective
+# rubocop:disable Metrics/PerceivedComplexity
+# Triangle analyzes the lengths of the sides of a triangle
+# (represented by a, b and c) and returns the type of triangle.
+#
+# It returns:
+#   :equilateral  if all sides are equal
+#   :isosceles    if exactly 2 sides are equal
+#   :scalene      if no sides are equal
+#
+# The tests for this method can be found in
+#   about_triangle_project.rb
+# and
+#   about_triangle_project_2.rb
+#
+def triangle(sd1, sd2, sd3)
```

Comment:
sd1? Does it mean Software Development or Secure Digital maybe?

Reply:
Super-Duper (just joking). It's not a big deal here to have full `side1, side2, side3`